### ISSUE TITLE
[Detail View Subtitle] Insufficient Color Contrast

### ISSUE THREAD

**Issue by laurunning** [cerner/terra-clinical#245]
# Issue Description

Issue: the subtitle gray text does not pass the accessibility WCAG AA color contrast ratio: 

![image](https://user-images.githubusercontent.com/32143728/34616738-b8d9ee4a-f1fe-11e7-80d8-8c445986b9a3.png)

![image](https://user-images.githubusercontent.com/32143728/34616566-4d5e091c-f1fe-11e7-886e-16653754ffce.png)
contrast between background (#ffffff) and Subtext text color (#BCBFC0).
## Issue Type

- [ ] New Feature
- [x] Enhancement
- [ ] Bug
- [ ] Other

## Current Behavior

The subtitle gray color of the detail view does not pass the WCAG AA contrast ratio.

## Steps to Reproduce

1. 
2.
3.
4.

## Environment

* Component Version:
* Browser Name and Version: Chrome 
* Operating System and version (desktop or mobile): OS desktop

**Comment by neilpfeiffer** [comment]
Will be resolved by PR #297
